modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/config/StorageTarget.scala

```diff
@@ -341,6 +340,16 @@ object StorageTarget {
   implicit def parameterStoreConfigDecoder: Decoder[ParameterStoreConfig] =
     deriveDecoder[ParameterStoreConfig]
 
+  implicit def nullableCronExprDecoder: Decoder[Option[CronExpr]] =   Decoder.instance { cur =>
```

Comment:
I understand that this nullable decoder allows you to write `"analyzeEventsSchedule": ""` to cancel the schedule.  And I understand you did it because the default value in `application.conf` is non-null.

However, is it also possible to put this in a hocon file:

```
analyzeEnventsSchedule: null
```

[This documentation](https://github.com/lightbend/config/blob/main/HOCON.md) hints that it might be possible, but I've never tried it.

Reply:
I find it more user-friendly to allow "". There is no drawback to allowing that, null will still work.